probe-rs/src/architecture/arm/memory/adi_v5_memory_interface.rs

```diff
@@ -48,26 +48,6 @@ where
 
 impl<APA> ADIMemoryInterface<'_, APA> where APA: ApAccess {}
 
-impl<APA> SwdSequence for ADIMemoryInterface<'_, APA>
```

Comment:
Wouldn't hurt to keep this one either way, no?

Reply:
I think it's clearer if code just uses the lower level interface directly.